invenio_accounts/api.py

```diff
@@ -8,25 +8,60 @@
 
 """API objects for Invenio Accounts."""
 
-from collections import defaultdict
+
+class Session:
+    """Session object for DB Users change history."""
+
+    def __init__(self):
+        self.updated_users = []
```

Comment:
(unless I am missing something) all the operations performed on this attributes are additions and check for existance (`is in...`). In that case, a set() would perform better. WDYT?